src/components/picker/api/picker.api.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "Picker",
+  "category": "form",
+  "description": "Picker Component, support single or multiple selection, blurModel and nativePicker",
```

Comment:
```suggestion
  "description": "Picker Component, support single or multiple selection, blurModel and native wheel picker",
```

Reply:
I'll fix in the original description as well